code/modules/virus2/effect/stage_4.dm

```diff
@@ -913,13 +913,12 @@
 	name = "Forced Humanity Syndrome"
 	desc = "A recent development by human supremacists. Causes non-human infected to mutate into a Human."
 	stage = 4
-	badness = EFFECT_DANGER_DEADLY
+	badness = EFFECT_DANGER_HELPFUL
```

Comment:
🤔

Reply:
Darn it, he looked.

Seriously though, it doesn't have any business being at deadly.